src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/ProjectSystem/RazorConnectHandler.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts;
+using Microsoft.CodeAnalysis.Razor.Protocol;
+using Microsoft.CommonLanguageServerProtocol.Framework;
+using Microsoft.VisualStudio.Threading;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.ProjectSystem;
+
+[RazorLanguageServerEndpoint(CustomMessageNames.RazorConnectEndpointName)]
+internal class RazorConnectHandler(NamedPipeBasedRazorProjectInfoDriver infoDriver) : IRazorNotificationHandler<RazorConnectParams>
```

Comment:
```suggestion
internal sealed class RazorConnectHandler(NamedPipeBasedRazorProjectInfoDriver infoDriver) : IRazorNotificationHandler<RazorConnectParams>
```

Reply:
Need an analyzer for this 😅 (I bet there is one)